### DISCUSSION TITLE
How to reconfigure security onion 2.3.40 nginx url from IP address to FQDN?

### DISCUSSION THREAD

**Discussion by sajeevka** [Security-Onion-Solutions/securityonion#3827] [Unsupported Versions]
I initially configured the server with an IP address and now I have a fqdn and TLS certs. how to make this change in Security Onion 2.3.40?

**Comment by weslambert** [comment]
You need to replace the `managerssl` cert/key in `/etc/pki` with your own.  Also, you'll want to modify url_base in `/opt/so/saltstack/local/pillar/global.sls` and restart nginx/SOC.
